sklearn/metrics/tests/test_common.py

```diff
@@ -331,15 +336,20 @@ def precision_recall_curve_padded_thresholds(*args, **kwargs):
     "hamming_loss",
 
     "precision_score", "recall_score", "f1_score", "f2_score", "f0.5_score",
+    "jaccard_score",
```

Comment:
jaccard score is missing is some sets.
e.g., ``jaccard_score`` should in ``METRIC_UNDEFINED_MULTICLASS``, ``jaccard_score`` should in ``METRICS_WITH_POS_LABEL``.
My suggestion is to search ``recall_score`` and add ``jaccard_score`` if needed.

Reply:
Thanks